### PR TITLE
New register benefits

### LINKED ISSUE #371: Refactor MembershipPlan Component
Looking at other membership plan selection pages across the internet, we see:

Netflix:
image

This example:
image

And this example:
image

Here's our current website:
image

Gross! Notice how we use two obscure photos and then copy and paste the text across both plans. With that being said, we could do an approach like:
image

### How to do this
1. Realize the code is found in [src/Pages/MembershipApplication/MembershipPlan.js](https://github.com/SCE-Development/Core-v4/blob/dev/src/Pages/MembershipApplication/MembershipPlan.js)
1. Start by creating a component to render the benefits we offer with any emojis you feel fit thrown in
1. Refactor [this `<div>` block](https://github.com/SCE-Development/Core-v4/blob/e8aa1a475258b750b10f52f53931a2ca0659cc18/src/Pages/MembershipApplication/MembershipPlan.js#L102) to show membership plan, price and expiration date

Done!

### PR DESCRIPTION
**Pull request by Annabel1054**
Resolves #371 
image

### CODE REVIEW

**Comment by Annabel1054** [comment]
image

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 1]
```diff
@@ -45,7 +49,7 @@ class MembershipPlan extends Component {
       plan.includes('Fall ' + this.state.year) ||
       plan.includes('Spring and Fall ' + this.state.year)
     ) {
-      return `December 20th, ${this.state.year}`;
+      return `Dec 20th, ${this.state.year}`;
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:52]:
```suggestion
      return `Dec. 20th, ${this.state.year}`;
```

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 2]
```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
...
+        <Container id='container-membership-plan'>
+          <h1 className='greet' style={{ color: 'white' }}>Welcome</h1>
+
+          <div className='card' >
+            <div className='membership'>
+              <Container className='top-chunk'>
+                <Row className='benefit-block grid'>
+                  <Col className='col'>
+                    <div className='how-to-join-block'>HOW TO JOIN:
+                      <p>1️⃣ SELECT a plan below</p>
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:102]: consider doing `array.map` here, we may need to change the steps someday and using an array is scalable

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 3]
```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
...
+                      <p>2️⃣ CLICK on "Add Account Information"</p>
+                      <p>3️⃣ Sign Up @ SCE (ENGR 294)</p>
+
+                    </div>
+                  </Col>
+                  <Col className='col'>
+                    <div>
+                      <h3 className='why-join-block'>WHY JOIN?
+                        <p>🌱 Free Paper Printing 🖨️ Free 3D Printing</p>
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:111]: consider `array.map` here as well, as benefits may change one day.

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 4]
```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
...
+                <Button className='add-acc' style={{ marginBottom: '30px' }}
+                  disabled={!this.state.planSelected}
+                  onClick={() =>
+                    this.props
+                      .setMembershipState(memberApplicationState.FORM_INFO)
+                  }
+
+                >
+                  Add Account Information
+
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:153]: remove blank line

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 5]
```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
...
+                </Button>
+              </Row>
+
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:156]: remove blank line

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 6]
```diff
@@ -83,45 +89,76 @@ class MembershipPlan extends Component {
...
+            </div>
+          </div>
+
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:159]: remove blank line

File: src/Pages/MembershipApplication/register-page.css [hunk 7]
```diff
@@ -80,14 +81,158 @@
   transition: opacity 200ms ease-in;
 }
 
-#change-and-select-btns {
+#change-and-select-btns .membership-price {
   background-color: #e1f5fe;
   border-color: black;
   color: black;
-}
+}
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/register-page.css:88]: remove trailing space

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 8]
```diff
@@ -21,7 +22,34 @@ class MembershipPlan extends Component {
       activeId: undefined,
       planSelected: false,
       year: new Date().getFullYear(),
-      planType: []
+      planType: [],
+      instructions: ['1️⃣ SELECT a plan below',
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/MembershipPlan.js:26]: delete lines 26 to 33. You have that info in blocks[]

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 9]
```diff
@@ -83,45 +113,63 @@ class MembershipPlan extends Component {
...
-                  type.plan === this.state.activeId
-                    ? 'card membership-card active-plan'
-                    : 'card membership-card'
-                }
-                id={type.plan}
-                key={ind}
-                onClick={this.cardSelected.bind(this, type.plan)}
+      <div className='background'>
+        <Container id='container-membership-plan'>
+          <h1 className='greet' style={{ color: 'white' }}>Welcome</h1>
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/MembershipPlan.js:118]: move inline styling to css

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 10]
```diff
@@ -83,45 +113,63 @@ class MembershipPlan extends Component {
...
+          <div className='card' >
+            <div className='membership'>
+              <Container className='top-chunk'>
+                <Row className='benefit-block grid'>
+
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/MembershipPlan.js:123]: delete line

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 11]
```diff
@@ -83,45 +113,63 @@ class MembershipPlan extends Component {
...
+                  {this.state.blocks.map((type, index) => (
+                    <Col key={index} className='col'>
+                      <div className={type.className}>{type.title}
+                        {type.details.map((text, index) =>
+                          (<p key={index}>{text}</p>))}
+                      </div>
+                    </Col>
+                  ))
+                  }
+
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/MembershipPlan.js:133]: delete empty line

File: src/Pages/MembershipApplication/register-page.css [hunk 12]
```diff
@@ -80,14 +81,157 @@
...
+  font-size: large;
+  background-color: transparent;
+  min-width: 400px;
+}
+
+/* Add some padding inside the card container */
+.container {
+  padding: 2px 0px;
+  text-align: center;
+
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/register-page.css:112]: delet empty line

File: src/Pages/MembershipApplication/register-page.css [hunk 13]
```diff
@@ -80,14 +81,157 @@
...
+  align-items: center;
+  text-align: center;
+}
+
+.q {
+  align-items: center;
+  text-align: center;
+}
+
+/* .expiration {
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/register-page.css:125]: delete commented code

File: src/Pages/MembershipApplication/register-page.css [hunk 14]
```diff
@@ -80,14 +81,157 @@
...
+  gap: 3.5rem;
+  text-align: center;
+  grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+}
+
+.col {
+  margin: 0px !important;
+  padding: 0px
+}
+
```
Comment (IsitaB) [inline src/Pages/MembershipApplication/register-page.css:237]: the file's formatting is inconsistent. Could you please format it cmd-opt-F

File: src/Pages/MembershipApplication/MembershipPlan.css [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+    padding: 30px;
+    padding-top: 0px;
+    width: 100%;
+    height: 100%;
+    min-width: 40%;
+    text-align: center;
+}
+
+.grid {
+    /* grid-template-columns: repeat(auto-fill, minmax(350px, 1fr)); */
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.css:201]: remove this

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 16]
```diff
@@ -21,7 +21,26 @@ class MembershipPlan extends Component {
       activeId: undefined,
       planSelected: false,
       year: new Date().getFullYear(),
-      planType: []
+      planType: [],
+      blocks: [
+        {
+          className: 'how-to-join-block',
+          title: 'HOW TO JOIN',
+          details: ['1️⃣ SELECT a plan below',
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:29]:
```suggestion
          details: [
            '1️⃣ SELECT a plan below',
```

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 17]
```diff
@@ -21,7 +21,26 @@ class MembershipPlan extends Component {
...
+            '2️⃣ CLICK on "Add Account Information"',
+            '3️⃣ Sign Up @ SCE (ENGR 294)'],
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:31]:
```suggestion
            '3️⃣ Sign Up @ SCE (ENGR 294)'
          ],
```

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 18]
```diff
@@ -21,7 +21,26 @@ class MembershipPlan extends Component {
...
+        },
+        {
+          className: 'why-join-block',
+          title: 'WHY JOIN',
+          details: [
+            '🌱 Free Paper Printing 🖨️ Free 3D Printing',
+            '🏢 Company Tours 💯 Access to Workshops',
+            '👕 Free Club T-Shirt 🏠 Free Door Code',
+            'Club Bonding Events: Potlucks, Game Nights'],
```
Comment (evanugarte) [inline src/Pages/MembershipApplication/MembershipPlan.js:40]:
```suggestion
            'Club Bonding Events: Potlucks, Game Nights'
          ],
```

**Pull request merged**